model/fieldtypes/DBFloat.php

```diff
@@ -60,3 +60,7 @@ public function prepValueForDB($value) {
 	}
 
 }
+
+if (PHP_MAJOR_VERSION < 7) {
+	include_once 'compat/Float.php';
```

Comment:
With a hammer-proof helmet on I present this solution.


```
<?php

if (PHP_MAJOR_VERSION < 7) {
    eval('class Int extends DBInt {}');
}
```

Still requires inclusion somewhere, as class manifest won't pick it up.